prettyprinter-configurable/src/Text/PrettyBy/Internal.hs

```diff
@@ -574,256 +617,339 @@ application is implemented.
 
 -- | Throw @err@ when @b@ is stuck.
 type family ThrowOnStuck err (b :: Bool) :: Bool where
-    ThrowOnStuck _   'True  = 'True
-    ThrowOnStuck _   'False = 'False
-    ThrowOnStuck err _      = err
+  ThrowOnStuck _ 'True = 'True
+  ThrowOnStuck _ 'False = 'False
+  ThrowOnStuck err _ = err
```

Comment:
Huh, this won't conflict with `stylish-haskell`?

Reply:
It probably does. My understanding is that we're going to stop using stylish haskell once we have adopted fourmolu.